core/src/main/java/com/scalar/db/sql/statement/BindableStatement.java

```diff
@@ -0,0 +1,11 @@
+package com.scalar.db.sql.statement;
+
+import com.scalar.db.sql.Value;
+import java.util.List;
+import java.util.Map;
+
+public interface BindableStatement<T extends Statement> extends Statement {
```

Comment:
We can bind parameters to the statement classes that implement this `BindableStatement`.